Makefile

```diff
@@ -1,6 +1,9 @@
-# minimal Makefile to run Python unit tests
-# for flux-accounting
+FLUX_INSTALL_PREFIX ?= /usr
+
+install:
+	${FLUX_INSTALL_PREFIX}bin/flux python -m pip install -r requirements.txt --user
```

Comment:
It might be worth splitting this dependency installation step out into a separate target:

```make
dependencies:
	${FLUX_INSTALL_PREFIX}bin/flux python -m pip install -r requirements.txt --user
```

And then you can make that auto-run during a `make install` with:

```make
install: dependencies
      INSTALL_CMDS_GO_HERE
```

That way, if users want to run the testsuite without installing flux-accounting but don't have the dependencies installed, they can do:

```
make dependencies
make check
```

Reply:
TIL! Thanks for explaining this - this is a neat feature.